api/pb/api.proto

```diff
@@ -140,6 +140,58 @@ message RemoveAppTokenRequest {
 
 message RemoveAppTokenReply {}
 
+message AddFileRequest {
```

Comment:
I changed from "store" to "add files", which feels more intuitive on the command line as well.